web/src/i18n/locales/en.json

```
@@ -134,6 +134,7 @@
         "messageRequired": "A brief message is required",
         "URLValid": "URL must be valid",
```

I am noticing that this message is a bit vague if the user forgets to start with http/https://

Perhaps 
```suggestion
        "URLValid": "Invalid URL: Please begin with http:// or https://",
```

---

@ann-kilzer done!